packages/@aws-cdk/aws-eks/lib/kubectl-handler/apply/__init__.py

```diff
@@ -24,6 +24,7 @@ def apply_handler(event, context):
     role_arn      = props['RoleArn']
     prune_label   = props.get('PruneLabel', None)
     overwrite     = props.get('Overwrite', False)
+    validation_flag = props.get('ValidationFlag', None)
```

Comment:
Why do we want to default it to `False`?

Reply:
Whoops sorry, good catch! `True` then :)

Reply:
I think we can leave it as `None` since that will skip the flag altogether and take whatever `kubectl` considers as default.

Reply:
Yeah I see why you named it `validation_flag` now. The thing is that if we say the default is "whatever kubernetes does when not specifying --validate", than we should advertise that, instead of stating that validation is enabled by default. 

Otherwise we should pass `--validate=true` as the default.

Im also good with changing it to `skipValidation: true` where the default would be `false`, meaning that we let kubernetes do its thing, and then using `None` and `False` here would be exactly the same.

I know it a little pedantic, but its a source of confusion and possible a pretty nasty unexpected behavior in the future.

Reply:
Yeah I understood the concern now. I think `skipValidation` seems like a better way to handle this. Let me make the change.